### DISCUSSION TITLE
You can use 🎷 symbols when naming playlists

### DISCUSSION THREAD

**Discussion by StrangeSeasOfThought** [Taiko2k/Tauon#1005] [Show and tell]
I visited the themes thread and noticed that no one is using any symbols. I love using them because they change the look and feel in a small but meaningful way. There are hundreds of symbols available that can be used.

Tip: For smaller playlist tabs/to save space you can simply use a symbol without any text.

![symbols1](https://github.com/Taiko2k/TauonMusicBox/assets/43582772/1279dbbd-d6ba-4311-97e8-889b7732e8d7)

![symbols2](https://github.com/Taiko2k/TauonMusicBox/assets/43582772/2a70045d-4d6f-4001-9bef-2817982a5d1b)

**Comment by Solarunit** [comment]
interesting, I guess I just prefer more minimal look. On your screenshot icons are different color and shape, maybe if they were the same style and one color palette.. :thinking:
